csharp/src/Apache.Arrow/Ipc/ArrowTypeFlatbufferBuilder.cs

```diff
@@ -171,6 +172,13 @@ public void Visit(Time64Type type)
                     Flatbuf.Time.CreateTime(Builder, ToFlatBuffer(type.Unit), 64));
             }
 
+            public void Visit(StructType type)
+            {
+                Flatbuf.Struct_.StartStruct_(Builder);
+                FieldType result = FieldType.Build(Flatbuf.Type.Struct_, Flatbuf.Struct_.EndStruct_(Builder));
```

Comment:
(super nit) Do you need the `FieldType result` variable? Why not just assign `Result` directly, like the other methods do?